packages/fiori/src/NotificationListGroupItem.ts

```diff
@@ -157,7 +146,7 @@ class NotificationListGroupItem extends NotificationListItemBase {
 	 */
 	clearChildBusyIndicator() {
 		this.items.forEach(item => {
-			item.busy = false;
+		  item.busy = false;
```

Comment:
probably the change is not needed, unless it fixes the identation

Reply:
Wow, I dont know how that happend. Thanks !